src/Components/Artwork/Details.tsx

```diff
@@ -132,6 +135,17 @@ const SaleInfoLine: React.FC<DetailsProps> = props => {
   )
 }
 
+const HighDemandInfo = () => {
+  return (
+    <Flex flexDirection="row">
+      <HighDemandIcon />
+      <Text variant="sm-display" color="blue100" ml={0.5}>
```

Comment:
**Suggestion:** the variant according to the design is `xs` 🌚 
```suggestion
      <Text variant="xs" color="blue100" ml={0.5}>
```

Reply:
Thank youuuu! 🙏